server/index.js

```diff
@@ -17,6 +17,22 @@ app.get('/:pid/recommendation/getInfo', (req, res) => {
   });
 });
 
+app.get('/:pid', (req, res) => {
```

Comment:
Are you planning to edit the endpoint later on? For example, product/:pid ?

Reply:
Maybe I should, that would look for realistic.